src/functions/getTestResultsByTesterStaffId.ts

```diff
@@ -10,13 +10,13 @@ https://github.com/aws/aws-xray-sdk-node/issues/14
 /* tslint:disable */
 let AWS:any;
 if (process.env._X_AMZN_TRACE_ID) {
-  AWS = require("aws-xray-sdk").captureAWS(require("aws-sdk"));
+  /* tslint:disable */
+   AWS = require("aws-xray-sdk");
```

Comment:
Does that works?? Can't imagine the XRAY sdk has  all the methods of the original

Reply:
Yes it is. Tested during regression.

Reply:
This is only because you don't actually use AWS anywhere, only XRay. Better to simply import XRay without any of the "if" logic, and rely on the if statement on line 22  wrapping your calls for safety.